snowshu/adapters/target_adapters/base_target_adapter.py

```diff
@@ -125,16 +126,20 @@ def load_data_into_relation(self, relation: Relation) -> None:
             raise exc
         logger.info('Data loaded into relation %s', relation.quoted_dot_notation)
 
-    def initialize_replica(self, source_adapter_name: str) -> None:
+    def initialize_replica(self, 
+                           source_adapter_name: str,
+                           target: str) -> None:
```

Comment:
Since we use "source" and "target" in a bunch of different circumstances, we should make sure this arg (and probably most other references) indicates that it's the target docker image as opposed to some other target type.